src/index.html

```diff
@@ -2,13 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav">
+      <div class="nav__wrapper">
+        <div class="nav__logo">
+          <img src="images/logo.png" alt="moyo logo">
+        </div>
+        <nav class="nav__nav">
+          <ul class="nav__list">
+            <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
```

Comment:
It would be great to format your code as follow to make it easier to read:
```suggestion
            <li class="nav__item">
              <a href="#" class="nav__link is-active">
                Apple
              </a>
            </li>
```
Fix all cases
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/7c51f357-65a9-48a4-aca7-78f69687dc09)